src/Accelerators/Accelerator.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===-------------------------- OMAccelerator.hpp -------------------------===//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// OMAccelerator base class
+//
+//===----------------------------------------------------------------------===//
+#include "src/Accelerators/Accelerator.hpp"
+#include "src/Accelerators/NNPA/NNPAAccelerator.hpp"
+#include <iostream>
+#include <vector>
+
+namespace mlir {
+extern NNPAAccelerator nnpaAccelerator;
```

Comment:
Is it necessary to have `nnpa` anything here? (include, variable)

Reply:
Yep. It needs to be here to get the accelerator included.